Project.toml

```diff
@@ -1,6 +1,6 @@
 name = "LRUCache"
 uuid = "8ac3fa9e-de4c-5943-b1dc-09c6b5f20637"
-version = "1.4.1"
+version = "1.5.0"
```

Comment:
counted this as a "feature", but I think it could also be seen as a bugfix (and thus a patch release)